pkg/controller/machine/machine_controller.go

```diff
@@ -221,17 +236,77 @@ func (r *ReconcileMachine) deleteNode(ctx context.Context, cluster *clusterv1.Cl
 	// Otherwise, proceed to get the remote cluster client and get the Node.
 	remoteClient, err := remote.NewClusterClient(r.Client, cluster)
 	if err != nil {
-		klog.Errorf("Error creating a remote client for cluster %q while deleting Machine %q, won't retry: %v",
+		klog.Errorf("error creating a remote client for cluster %q while deleting Machine %q, won't retry: %v",
```

Comment:
why lower case?

Reply:
@vincepri asked me to make it lowercase.

Reply:
@vincepri what's the convention here? Generally if a message is just printed use uppercase, if the message is returned and passed on, lowercase

Reply:
Apologies for the confusion:
- klog (or general log, including warnings and errors) messages start with an uppercase letter
- Errors start with a lowercase